readetexts-port.patch

```diff
@@ -200,11 +228,20 @@ index b623d59..5de0af0 100644
          self.alert(_('Success'), self.selected_title + _(' added to Journal.'))
  
      def find_previous(self):
+@@ -1366,7 +1327,7 @@ class ReadEtextsActivity(activity.Activity):
+         self.current_found_item = -1
+         self.etext_file.seek(0)
+         while self.etext_file:
+-            line = str(self.etext_file.readline())
++            line = str(self.etext_file.readline().decode('iso-8859-1'))
```

Comment:
Please move everything related to this fix to a separate patch, so it's easier to rebase next time.